cookbook/case_studies/in_container.mk

```diff
@@ -0,0 +1,24 @@
+SERIALIZED_PB_OUTPUT_DIR := /tmp/output
```

Comment:
Is this a copy from `cookbook/in_container.mk`?

Reply:
yeah :( any ideas for getting rid of that? as a follow up PR probably...